datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py

```diff
@@ -32,9 +33,19 @@
         'not be considered.'
     )
 )
+@click.option(
+    '--force', '-f', 'force',
```

Comment:
You can omit the force on the end. Specifying the variable name is mostly for conflicts with built-ins, like `all`.